peekingduck/pipeline/nodes/model/yolov4/yolo_files/detector.py

```diff
@@ -128,7 +128,7 @@ def _evaluate_image_by_yolo(
             - nums: number of valid bboxes. Only nums[0] should be used. The rest
                     are paddings.
         """
-        # image = image[..., ::-1]  # swap from bgr to rgb
+        image = image[..., ::-1]  # swap from bgr to rgb
```

Comment:
Consider using `cv2.cvtColor` so that the code is self explanatory (with help of opencv's conversion code) without having to resort to comments